src/BMPDecoder.ts

```diff
@@ -22,44 +26,103 @@ export default class BMPDecoder {
     this.width = this.bufferData.skip(4).readUint32();
     this.height = this.bufferData.readUint32();
     this.bitDepth = this.bufferData.seek(28).readUint16();
+    this.compression = this.bufferData.readUint32();
+    this.imageSize = this.bufferData.readUint32();
     this.xPixelsPerMeter = this.bufferData.seek(38).readInt32();
     this.yPixelsPerMeter = this.bufferData.readInt32();
-    if (this.bitDepth !== 1) {
-      throw new Error('only bitDepth of 1 is supported');
-    }
+    this.logicalColorSpace = this.bufferData.readUint32();
+    this.colorMasks = [
```

Comment:
Since you only support the "standard" RGB(A) mask, you should throw an error if it is not in the supported range.